### ISSUE TITLE
`color(colorString, alpha)` produces console error

### ISSUE THREAD

**Issue by ajrussellaudio** [processing/p5.js#2779]
#### Nature of issue?

- [x] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [x] Color
- [ ] Core
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [x] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: **0.6.0 loaded from Cloudflare CDN**
- Web browser and version: **Chrome 65.0.3325.181 (Official Build) (64-bit)**
- Operating System: **macOS 10.12.6**
- Steps to reproduce this:
```js
// sketch.js
function setup() {
  var trasparentRed = color("#FF0000", 128);
}
```
produces the Chrome console error:
```
Uncaught Error: [object Arguments]is not a valid color representation.
    at p5.Color._parseInputs (p5.js:43792)
    at new p5.Color (p5.js:43004)
    at p5.color (p5.js:42683)
    at setup (sketch.js:2)
    at p5.<anonymous> (p5.js:46215)
    at p5.<anonymous> (p5.js:46150)
    at new p5 (p5.js:46434)
    at _globalInit (p5.js:48404)
```

The [reference docs](https://p5js.org/reference/#/p5/color) suggest this should work, as in `color(value,[alpha])` where `value` is a color String.

**Comment by lmccart** [comment]
This is not supported.

The docs say:
"If a single string argument is provided, RGB, RGBA and Hex CSS color strings and all named color strings are supported. In this case, an alpha number value as a second argument is not supported, the RGBA form should be used."

Is there something you're seeing in the reference that contradicts this?

**Comment by Spongman** [comment]
the bug is in the syntax section:

```javascript
/**
  * @method color	  * @method color
  * @param  {String}        value   a color string	  * @param  {String}        value   a color string
- * @param  {Number}        [alpha]	
  * @return {p5.Color}	  * @return {p5.Color}
  */
```

that overload is invalid - it shouldn't take an alpha value.

i included a fix for this (and an additional test) in #2592.

**Comment by lmccart** [comment]
ah ok i see thanks. i will try to review that pr as soon as i can. it is difficult because there is so much in there. it would be very helpful if you could limit each pr to addressing one issue at a time.

**Comment by Spongman** [comment]
indeed. i tried to keep it small, but it's all quite inter-dependent. triggering the test failures requires updates to the tests, and to the FES checker to make it more robust.
